static/js/ui.js

```diff
@@ -13,34 +15,37 @@ exports.replace_emoji_with_text = function (element) {
     });
 };
 
-exports.set_up_scrollbar = function (element) {
-    var perfectScrollbar = new PerfectScrollbar(element[0], {
-        suppressScrollX: true,
-        useKeyboard: false,
-        wheelSpeed: 0.68,
-        scrollingThreshold: 50,
-        minScrollbarLength: 40,
-    });
-    element[0].perfectScrollbar = perfectScrollbar;
+exports.set_up_scrollbar = function (element_selector) {
+    new SimpleBar(element_selector.expectOne()[0]);
+};
+
+exports.get_scroll_element = function (element_selector) {
+    var element = element_selector.expectOne()[0];
+    if (element.SimpleBar) {
+        return $(element.SimpleBar.getScrollElement());
+    }
+    return element_selector;
 };
 
 exports.update_scrollbar = function (element_selector) {
-    var element = element_selector[0];
-    if (element.perfectScrollbar !== undefined) {
-        element.perfectScrollbar.update();
+    var element = element_selector.expectOne()[0];
+    if (element.SimpleBar) {
+        element.SimpleBar.recalculate();
     }
 };
 
 exports.reset_scrollbar = function (element_selector) {
-    var element = element_selector[0];
-    element.scrollTop = 0;
-    if (element.perfectScrollbar !== undefined) {
-        element.perfectScrollbar.update();
+    var element = element_selector.expectOne()[0];
+    if (element.SimpleBar) {
+        element.SimpleBar.getScrollElement().scrollTop = 0;
+        element.SimpleBar.recalculate();
+    } else {
+        element.scrollTop = 0;
```

Comment:
When does this else clause come up?  Or is this just trying to port the existing logic?

Reply:
Porting existing logic, yes, but it comes up opening “Manage streams” for some reason.